proteus/NumericalSolution.py

```diff
@@ -255,6 +255,7 @@ def __init__(self,so,pList,nList,sList,opts,simFlagsList=None):
                     mlMesh = MeshTools.MultilevelTriangularMesh(0,0,0,skipInit=True,
                                                                 nLayersOfOverlap=n.nLayersOfOverlapForParallel,
                                                                 parallelPartitioningType=n.parallelPartitioningType)
+                    logEvent("NAHeader GridRefinements %i" % (n.nLevels) )
```

Comment:
Yes, NAHeader is one of the text flags I use to extract data for the residual analysis.  That said, I don't want to pollute the output unnecessarily for other users.  Perhaps I should set the log-levels for these rather high or create a separate profiling method entirely?